packages/formatter-summary/src/formatter.ts

```diff
@@ -43,7 +42,7 @@ export default class SummaryFormatter implements IFormatter {
         }
 
         const buildMessage = (count, type) => {
-            return `${count} ${pluralize(type, count)}`;
+            return `${count} ${count !== 1 ? `${type}s` : type}`;
```

Comment:
I don't like the idea of template literals inside template literals in this case. I think the following is easier to understand:

```js
${count} ${type}${count === 1 : '' ? 's'}
```

Reply:
Done.